packages/main/src/plugin/configuration-registry.spec.ts

```diff
@@ -273,3 +273,20 @@ test('addConfigurationEnum with a previous default value', async () => {
   const val = configurationRegistry.getConfiguration('my.fake')?.get<string>('enum.property');
   expect(val).toEqual('myValue1');
 });
+
+describe('should be notified when a configuration is updated', async () => {
+  test('expect correct properties', async () => {
+    let called = false;
+    let updatedProperties: string[] = [];
+    configurationRegistry.onDidUpdateConfiguration(e => {
+      called = true;
+      updatedProperties = e.properties;
+    });
```

Comment:
```suggestion
    const subscriberMock = vi.fn();
    configurationRegistry.onDidUpdateConfiguration(subscriberMock);
    
    ...
    
    expect(subscriberMock).toHaveBeenCalledWith(...)
```

Would this be simpler than having to create boolean and property object to check it has been called ?

Reply:
I updated the code, it was using a copy/paste for the previous test for onDidChangeConfiguration

Reply:
:+1: